CHANGELOG.md

```diff
@@ -140,6 +144,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - **CUMULUS-2351**
   - Inventory report no longer includes the File/Granule relation object in the okCountByGranules key of a report. The information is only included when a 'Granule Not Found' report is run.
 
+### Removed
+
+- **CUMULUS-2367**
+  - Removed `execution_cumulus_id` column from granules RDS schema and data type
+
 ## [v5.0.0] 2021-01-12
 
 ### BREAKING CHANGES
```

Comment:
Generic, possibly ignorant, question: 

If we're removing the translation from db->api, and given we're not *reading* From RDS in phase 1....  isn't there a breaking issue here for the granules/files migration script and/or how will the reconciliation tool handle RDS being in a different structure than dyanmoDB?

Reply:
>. isn't there a breaking issue here for the granules/files migration script 

If you mean that the granules/files migration script also has to be updated, you're right. And I actually intended to just do that in this PR but forgot, so I can add that unless you object.

> how will the reconciliation tool handle RDS being in a different structure than dyanmoDB?

It somewhat depends on how the reconciliation tool really ends up working, but in theory if we know how to translate one record structure to another, it doesn't seem like a huge hurdle. There is an inherent incompatiblity in that PG can now return multiple executions per granule versus one execution per granule in Dynamo, but it seems easy enough to massage the data types for comparison unless I'm missing something

Reply:
TODO:

- Make sure to update granules/files migration script

Reply:
f1a6636b45889d75339a0f0ec3323f7f84750971

Reply:
This update addresses the concern, however I have a comment re: the TODO in the implementation.   Resolving this conversation in favor of that one.